salt/example.top

```diff
@@ -1,11 +1,6 @@
 base:
     '*':
         - elife
+        - elife.docker
         - elife.nginx
```

Comment:
Nginx to be containerised later?

Reply:
In this case it becomes just a proxy providing HTTPS, since the main image is an `nginx` one. Where this responsibility ends up depends on where the container runs, maybe https://istio.io/docs/